src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemDisplayFactory.cs

```diff
@@ -4,23 +4,30 @@
 
 using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
 using Microsoft.CodeAnalysis.NavigateTo;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
 using Microsoft.VisualStudio.Language.NavigateTo.Interfaces;
+using Microsoft.VisualStudio.Utilities;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.NavigateTo
 {
     internal sealed class NavigateToItemDisplayFactory : INavigateToItemDisplayFactory
     {
         private readonly IThreadingContext _threadingContext;
+        private readonly IUIThreadOperationExecutor _threadOperationExecutor;
+        private readonly IAsynchronousOperationListener _asyncListener;
 
-        public NavigateToItemDisplayFactory(IThreadingContext threadingContext)
+        public NavigateToItemDisplayFactory(
```

Comment:
is this mef created?  It doesn't have any attributes. Trying to figure out what in the PR is passing in the new parameters

Reply:
It is not mef created.  The eventual args flow in from here: https://github.com/dotnet/roslyn/pull/59828/files#diff-5cb25d4d3928f87177c69b785b0af7532862d8a61a68cd0dbdc53f6d1c8ea061R48